WalletWasabi.Packager/ArgsProcessor.cs

```diff
@@ -1,86 +1,87 @@
+#nullable enable
+
 using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace WalletWasabi.Packager
 {
+	/// <summary>
+	/// Class for processing program's command line arguments.
+	/// </summary>
 	public class ArgsProcessor
 	{
-		public bool IsReduceOnionsMode(string[] args)
+		public ArgsProcessor(string[]? args)
 		{
-			bool getOnions = false;
-			if (args != null)
+			if (args == null)
```

Comment:
https://github.com/zkSNACKs/WalletWasabi/blob/master/CONTRIBUTING.md#null-check
```suggestion
			if (args is null)
```